features/steps/insights_results_aggregator.py

```diff
@@ -150,3 +157,26 @@ def perform_aggregator_database_migration(context, version):
 def perform_aggregator_database_migration_to_latest(context):
     """Perform aggregator database migration to latest version."""
     perform_aggregator_database_migration(context, "latest")
+
+
+@given("Insights Results Aggregator service is started in background")
+def start_insights_results_aggregator_in_background(context):
+    """Start Insights Results Aggregator service in background."""
+    process = subprocess.Popen(
+        [INSIGHTS_RESULTS_AGGREGATOR_BINARY],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+    )
+    # background process -> we can't communicate() with it
+
+    # time to breath
+    time.sleep(BREATH_TIME)
+
+    # check if process has been created
+    assert process is not None, "Process was not created!"
```

Comment:
Nitpick: Maybe this can be done before the time.sleep, so no need to wait if everything goes wrong